.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: dart
+script: ./tool/travis.sh
```

Comment:
If you don't need to do any sudo'ing (which you shouldn't), adding this line:

    sudo: false

will use containerized builds on travis - the builds will start up much faster.